python/lsst/rapid/analysis/utils.py

```diff
@@ -86,6 +86,12 @@ def argMax2d(array):
     return maxCoords[0], uniqueMaximum, maxCoords[1:]
 
 
+def dayObsIntToString(dayObs):
```

Comment:
Wasn't this function also in rubinTv?

Reply:
I see `translate_dayObs` in `obs.lsst.ingest`, which looks to do something very similar?

Reply:
`lsst.obs.lsst.ingest` is all gen2 code and will be removed.

Reply:
Yup, good shout, forgot about one, I've removed from RubinTV and imported this there.